pkg/listener/handler.go

```diff
@@ -247,6 +247,22 @@ func (h *Handler) handleNewBlock(ctx context.Context, b types.Block) error {
 	return nil
 }
 
+func (h *Handler) genPublishMessages(revertedBlocks, newBlocks []types.Block) []interface{} {
```

Comment:
This function ignores the reverted blocks? Why do we want to change the behavior?

Reply:
Because I need to match the message format with the current StreamSource service usage.
And as it was defined:
`1 block = 1 msg`
 => in this flow, we will not send reverted blocks. The msg stream will look like this: b1, b2, b3
if re-org happened at b2, next time send: b2', b3', b4'...
each `b` is one message

Reply:
It completely changes the behavior of the service, that's not good.

Reply:
@hiepnv90 @Neokun : what is the conclusion of this discussion?
As I understand, now 1 message still = many blocks (reverted + new), is it correct?

Reply:
It depends on the implementation of publisher, data team will implement a publisher to publish only new blocks with 1 block each message to queue.